.github/workflows/tests.yml

```diff
@@ -31,10 +31,21 @@ jobs:
     - name: Build docs
       run: tox -e docs
 
+  build-snap:
+    name: Build snap
+    runs-on: ubuntu-latest
+    needs: lint-unittests-docs
+    steps:
+    - uses: actions/checkout@v2
+    - uses: snapcore/action-build@v1
+      id: snapcraft
+    - name: Install snap
```

Comment:
Is this necessary? I think building it also verifies that it could be installed.

Reply:
I don't think the build process tests that it can be installed. This snap is simple but any snap interfaces or snap deps should be installed (e.g. install a specific core image if it is needed by the snap).

Reply:
Ack